k8s-alpha.yml

```diff
@@ -77,3 +77,46 @@ spec:
         - name: mediawiki-conf
           configMap:
             name: mediawiki-conf-1
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mediawiki-conf-1
+data:
+  CustomSettings.php: |
```

Comment:
What prompted you to move the config here? I usually find it less readable when multiple langs/styles are mixed (yaml and php in this case). Wouldn't it make more sense to have it separate?